pkg/proc/fncall.go

```diff
@@ -278,9 +278,11 @@ func funcCallArgFrame(fn *Function, actualArgs []*Variable, g *G, bi *BinaryInfo
 		formalArg := &formalArgs[i]
 		actualArg := actualArgs[i]
 
-		//TODO(aarzilli): only apply the escapeCheck to leaking parameters.
-		if err := escapeCheck(actualArg, formalArg.name, g); err != nil {
-			return nil, fmt.Errorf("can not pass %s to %s: %v", actualArg.Name, formalArg.name, err)
+		if checkEscape {
+			//TODO(aarzilli): only apply the escapeCheck to leaking parameters.
+			if err := escapeCheck(actualArg, formalArg.name, g); err != nil {
+				return nil, fmt.Errorf("can not pass %s to %s: %v", actualArg.Name, formalArg.name, err)
```

Comment:
Since we're here, let's make this error a bit more explicit. Something like: `("cannot use %s as argument %s in function %s: %v", ...)`

Reply:
Or specify the argument number.